src/mavedb/lib/score_sets.py

```diff
@@ -62,8 +62,8 @@ def search_score_sets(db: Session, owner: Optional[User], search: ScoreSetsSearc
                 ScoreSet.keyword_objs.any(func.lower(Keyword.text).icontains(lower_search_text)),
                 ScoreSet.target_genes.any(
                     TargetGene.target_sequence.has(
-                        TargetSequence.reference.has(
-                            func.lower(ReferenceGenome.organism_name).icontains(lower_search_text)
+                        TargetSequence.taxonomy.has(
+                            func.lower(Taxonomy.organism_name).contains(lower_search_text)
```

Comment:
I'm not sure you want to change icontains to contains here

Reply:
My branch may mix with previous codes. It's interesting that the above filters all used `contains`. I changed this to `icontains`.

Reply:
Wait. No, in main branch all of them are `contains`. I don't know which part modifies them in this PR. 
https://github.com/VariantEffect/mavedb-api/blob/main/src/mavedb/lib/score_sets.py

Reply:
you should be looking at branch release-2024.0.0, not main, in which it is icontains. this was a change made in https://github.com/VariantEffect/mavedb-api/commit/17f9671df662f8f25efa5a0f810e2354b19229c2

Reply:
Yes, you're correct.